pom.xml

```
@@ -82,6 +82,9 @@
 
         <!-- CVE-2021-44228 Log4Shell, CVE-2021-4104, CVE-2021-45105, CVE-2021-44832 -->
         <log4j.version>2.17.1</log4j.version>
+
+        <!-- Sonar Setting -->
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

I don't think it's required, but for good measure, can you add the sonar.organization also?
```suggestion
        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
        <sonar.organization>bf2fc6cc711aee1a0c2a</sonar.organization>
```

---

Sure!